js/ui/load_panel.js

```diff
@@ -110,6 +110,13 @@ const LoadPanel = Overlay.inherit({
         ]);
     },
 
+    _setDeprecatedOptions() {
+        this.callBase();
+        extend(this._deprecatedOptions, {
+            'elementAttr': { since: '21.2', alias: 'loadPanel.wrapperAttr' }
```

Comment:
```suggestion
            'elementAttr': { since: '21.2', alias: 'wrapperAttr' }
```

And change it in all such places plz

Reply:
done